internal/pkg/util/shell/interpreter/interpreter.go

```diff
@@ -110,6 +123,16 @@ func GetEnv(hc interp.HandlerContext) []string {
 	return environ
 }
 
+// LookupEnv returns the value of a specific exported environment variable within
+// the context of the shell interpreter.
+func LookupEnv(env string, hc interp.HandlerContext) string {
```

Comment:
Let's make this more like the function just above it,`GetEnv()`.   Change the name to `GetEnvVar()` and put the `hc` variable first.

Reply:
thanks! refactored.